lib/python/rose/meta_type.py

```diff
@@ -101,6 +101,18 @@ def is_valid(self, value):
         return [True, None]
 
 
+class PythonBooleanMetaType(MetaType):
+
```

Comment:
Well, the rest of the file doesn't do this and the class name is pretty much what'd go in the docstring - "metadata type for Python Boolean" - so unless you have particularly strong feelings that it should I'd say no.